### PR TITLE
Blocking callback

### PR DESCRIPTION
**Pull request by IcToxi**

### CODE REVIEW

File: pages/examples/blocking-callback-2.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+app.layout = html.Div(
+    [
+        graph := dcc.Graph(),
+        html.Br(),
+        slider := dcc.Slider(
+            min=years.min(),
+            max=years.max(),
+            value=years.max(),
+            step=years[1] - years[0],
+            marks={int(i): str(i) for i in years},
```
Comment (IcToxi) [inline pages/examples/blocking-callback-2.py:42]: There might be a bug here, if you pass in `np.int64`, the layout will crash without any error.

**Comment by IcToxi** [comment]
Hi @Coding-with-Adam,

Maybe you can consider reading the documentation to figure out this component written by Emil.😅 Or should I adjust the animation time a bit longer, and you'll see how it works.

**Comment by Coding-with-Adam** [comment]
No need. Let's leave the app the way it is for now. 
Let me look more into this.

**Comment by AnnMarieW** [comment]
Just doing some cleanup of old PRs and issues.  Thanks for the contribution, but unfortunately, using Dash Extensions is not compatible with this project.  Closing this now.

**Pull request closed without merging**